docs/data/date-pickers/time-range-picker/time-range-picker.md

```diff
@@ -0,0 +1,18 @@
+---
+product: date-pickers
+title: React Time Range Picker component
+components: TimeRangePicker, DesktopTimeRangePicker, MobileTimeRangePicker, StaticTimeRangePicker
+githubLabel: 'component: TimeRangePicker'
+packageName: '@mui/x-date-pickers'
+materialDesign: https://material.io/components/date-pickers
+---
+
+# Time Range Picker [<span class="plan-pro"></span>](https://mui.com/store/items/material-ui-pro/)
```

Comment:
Current capitalization policy

```suggestion
# Time range picker [<span class="plan-pro"></span>](https://mui.com/store/items/material-ui-pro/)
```

Reply:
I am fine with removing the capital letters
But aren't we using them in similar scenarios ?

https://mui.com/material-ui/material-icons/
https://mui.com/x/react-date-pickers/date-range-picker/#main-content